.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
Do I understand correctly, that Ubuntu-latest does not __yet__ have `bats`? In that case, I'd feel better about leaving `latest` and merge with a red CI. I can also run the tests locally.

I worry that I will forget to update the image, and it will become very stale.

Reply:
Yes, you're correct.

Due to the change in github actions, ubuntu 20.04 will be used if ubuntu-latest is specified. 
https://github.blog/changelog/2020-10-29-github-actions-ubuntu-latest-workflows-will-use-ubuntu-20-04/#:~:text=October%2029%2C%202020-,GitHub%20Actions%3A%20Ubuntu%2Dlatest%20workflows%20will%20use%20Ubuntu%2D20.04,preview%20mode%20since%20June%202.

But `bats` ppa repo doesn't have focal like below.
http://ppa.launchpad.net/duggan/bats/ubuntu/dists/

So we can just leave it and marge with a red CI or change CI to install bats without ppa repo.
If installing bats without ppa is preferred in CI, I can look at it in a different PR